scripts/a11y-testing.js

```diff
@@ -112,44 +118,48 @@ const printResult = result =>
       process.exit(1);
     }
   }
+  try {
+    const links = await docsPages(root, page);
 
-  const links = await docsPages(root, page);
+    for (const link of links) {
+      await page.goto(link);
 
-  for (const link of links) {
-    await page.goto(link);
+      const { violations } = await new AxePuppeteer(page)
+        .disableRules('color-contrast')
+        .exclude(['figure[role="figure"']) // excluding figure[role="figure"] the duplicatory role is there for ie11 support
+        .analyze();
```

Comment:
I did some tests and some extra research and we definitely shouldn't be adding focus to grid so let's change the config to ignore `grid`.

After this change, you can remove all of the `DataGrid` work you added to handle `tabIndex`. 

```suggestion
    const { violations } = await new AxePuppeteer(page)
      .configure({
        rules: [
          { id: 'color-contrast', enabled: false },
          {
            id: 'scrollable-region-focusable',
            matches: '[role="grid"]',
          },
        ],
      })
      .analyze();
```

I also removed the `figure[role="figure"]` check because we removed `role="figure"` from our `<figure>` elements recently.